site/content/blogs/release-v117.en.md

```diff
@@ -0,0 +1,48 @@
+
+Copilot v1.17 brings with it a brand-new features and several improvements:
+
+* **Tracing for Request-Driven Web Service:** With the release of AWS X-Ray tracing support for AWS App runner services, you can now add `observability.tracing: awsxray` in your Request-Driven Web Service's 
+  manifest to send your traces to AWS XRay. [See detailed section](./#send-your-request-driven-web-services-traces-to-aws-xray).
+* **Increased visibility for progress trackers:** [See detailed section](./#define-messages-filter-policy-in-publishsubscribe-service-architecture).
+* **Bug fix:** Remove color formatting of suggested pipeline names ([#3437](https://github.com/aws/copilot-cli/pull/3437)).
+
+## What’s AWS Copilot?
+
+The AWS Copilot CLI is a tool for developers to build, release, and operate production ready containerized applications on AWS.  
+From getting started, pushing to staging, and releasing to production, Copilot can help manage the entire lifecycle of your application development.
+At the foundation of Copilot is AWS CloudFormation, which enables you to provision infrastructure as code in a single operation.
+Copilot provides pre-defined CloudFormation templates and user-friendly workflows for different types of micro services creation and operation,
+enabling you to focus on developing your application, instead of writing deployment scripts.
+
+See the section [Overview](../docs/concepts/overview.en.md) for a more detailed introduction to AWS Copilot.
+
+## Send Your Request-Driven Web Service's Traces to AWS X-Ray
+_Contributed by [Wanxian Yang](https://github.com/Lou1415926/)_
+
+You can now send traces generated your Request-Driven Web Services to AWS X-Ray. This allows you to visualize the service map and traces 
+through CloudWatch console or AWS X-Ray console.
+
+To use this feature, your service needs to be first instrumented with 
+[AWS Distro for OpenTelemetry](https://aws.amazon.com/otel/?otel-blogs.sort-by=item.additionalFields.createdDate&otel-blogs.sort-order=desc).
+You can either [manually instrument](https://aws-otel.github.io/docs/getting-started/python-sdk/trace-manual-instr) your service, 
+or for a quicker, easier set up, [auto-instrument](https://aws-otel.github.io/docs/getting-started/python-sdk/trace-auto-instr) 
+your service through Dockerfile without changing application code.
+
+After instrumenting your service, simply modify your Request-Driven Web Service's manifest to include the [observability](../docs/manifest/rd-web-service.en.md#observability) configuration:
+```yaml
+observability:
+  tracing: awsxray
+```
+
+After running `copilot svc deploy`, traces generated by your service will get sent to AWS X-Ray, allowing you to measure
+the current state of your services easily.
+
+## What’s next?
+
+Download the new Copilot CLI version by following the link below and leave your feedback on [GitHub](https://github.com/aws/copilot-cli/) or our [Community Chat](https://github.com/aws/copilot-cli):
```

Comment:
is the second link supposed to go to Gitter? If so, can we change on the 1.16 blog also?